api/pkg/s3/abortmultipartupload.go

```diff
@@ -52,6 +52,8 @@ func (s *APIService) AbortMultipartUpload(request *restful.Request, response *re
 		response.WriteError(http.StatusInternalServerError, err)
 		return
 	}
-	log.Infof("Delete object %s successfully.", objectKey)
-	response.WriteEntity(res)
+	*/
```

Comment:
Please remove unused code and imports

Reply:
@nguptaopensds thanks for your review, i will remove them.